.gitignore

```diff
@@ -1,3 +1,8 @@
 _site/
 .DS_Store
 *.swo
+.idea/misc.xml
+.idea/modules.xml
+.idea/python3statement.github.io.iml
+.idea/vcs.xml
+.idea/workspace.xml
```

Comment:
Side note, usually you want to put those in your machine global gitignore.

Reply:
Ah!  I had no idea there was such a thing.  Thank you!